gdal/frmts/rmf/rmfdataset.cpp

```diff
@@ -52,6 +52,17 @@ static const char RMF_UnitsMM[] = "mm";
 constexpr double RMF_DEFAULT_SCALE = 10000.0;
 constexpr double RMF_DEFAULT_RESOLUTION = 100.0;
 
+/* -------------------------------------------------------------------- */
+/*  Note: Due to the fact that in the early versions of RMF             */
+/*  format the field of the iEPSGCode was marked as a 'reserved',       */
+/*  in the header on its place in many cases garbage values were writen.*/
+/*  Most of them can be weeded out by the minimum EPSG code value.      */
```

Comment:
what if the 'garbage' value is actually a valid EPSG code ? Isn't there a version number somewhere to know if that field is valid or not, or some other form of consistency check that can be done, for example with the importFromPanorama() method ?

Reply:
Unfortunately, the version number of the header did not change with the starting of using this field as EPSG code. 
I will try to rework so that importFromEpsg ran only if  importFromPanorama fails.

Reply:
I'd suggest to run both importFromEpsg and importFromPanorama and doing a few consistency check, at least that the projection method is the same for example. importFromEPSG() will bring you the EPSG code, which is quite valuable, even if importFromPanorama() succeeds

Reply:
I did before I read your comment.